src/modules/10-common/layouts/TrialLicenseBanner.tsx

```diff
@@ -292,7 +292,7 @@ export const TrialLicenseBanner = (): React.ReactElement => {
   }
 
   return (
-    <div className={cx(css.trialLicenseBanner, expiredClassName)}>
+    <div className={cx(css.trialLicenseBanner, expiredClassName)} data-banner-is-visible="true">
```

Comment:
Maybe be a wee bit more specific? And you probably don't need the value
```suggestion
    <div className={cx(css.trialLicenseBanner, expiredClassName)} data-trial-license-banner-visible>
```

Reply:
yeah fair, mind blown when I saw it working

Reply:
The power of attribute and sibling selectors in CSS!